forge-gui/res/cardsfolder/upcoming/drowner_of_truth_drowned_jungle.txt

```diff
@@ -0,0 +1,18 @@
+Name:Drowner of Truth
+ManaCost:5 GU GU
+Types:Creature Eldrazi
+PT:7/6
+K:Devoid
+T:Mode$ SpellCast | ValidCard$ Card.Self | ConditionManaSpent$ C | Execute$ TrigToken | TriggerDescription$ When you cast this spell, if {C} was spent to cast it, create two 0/1 colorless Eldrazi Spawn creature tokens with "Sacrifice this creature: Add {C}."
```

Comment:
ConditionManaSpent$ C  is not the correct param, any ideas ?

Reply:
try without "Condition"

Reply:
No succes

Reply:
Check out Desecrate Reality

Reply:
`CheckSVar$ CastSA>Count$Adamant_1.Colorless.1.0`

Reply:
Thank you, working now